libs/s25main/gameData/BuildingConsts.cpp

```diff
@@ -352,6 +364,9 @@ const helpers::MultiEnumArray<SmokeConst, Nation, BuildingType> BUILDING_SMOKE_C
     vikings[BuildingType::DonkeyBreeder] = SmokeConst(4, {-27, -40});
     vikings[BuildingType::Vineyard] = SmokeConst(1, {18, -48});
     vikings[BuildingType::Winery] = SmokeConst(1, {-14, -32});
+    vikings[BuildingType::Skinner] = SmokeConst(1, {7, -39});
```

Comment:
I made a typo with this one: it should be: vikings[BuildingType::Skinner] = SmokeConst(1, {-7, -39});

Reply:
done